### DISCUSSION TITLE
Do you know if this only gets the first 50 entries in your portfolio?

### DISCUSSION THREAD

**Discussion by ClaireCJS** [jessecooper/pyetrade#86] [General]
I was looking to see if you addressed the 50-entity limit on the eTrade api, because my personal etrade portfolio fetcher, which came from another github repo, only fetches 50 and doesn't send a "count" parameter to increase the limit past 50. I was looking to see how others addressed this.

**Comment by 1rocketdude** [comment]
are you asking about getting account positions? If so, this is what works for me

positions = accounts_api.get_account_portfolio(accountIdKey,'json',count=max_positions)

where max_positions=200 for example...

**Reply by ClaireCJS** [reply]
Oh very nice, thanks for the example!

My issues is the code I picked up from...somewhere...does it like this:

```
        url =self.base_url + "/v1/accounts/" + self.account["accountIdKey"] + "/portfolio.json"
        response = self.session.get(url, header_auth=True)
```

So the call isn't the same kind of call where I know how to insert count as a parameter. [Adding "?count=99" didn't work]

You wouldn't happen to know, would you?

It's an ugly situation, because I think if I switch to your way, the later code that processes the results would require modification that may be beyond my knowledge. Though I may have to do just that.  Or sell some positions lol.
